index.html.in

```diff
@@ -468,13 +468,13 @@
                             </div>
                         </div>
                         <div class="row">
-                            <div style="display: inline-block; margin: 0 .5em 0 .5em">
+                            <div class="display-inline halfEmMargin">
                                 <a href="http://creativecommons.org/licenses/by-sa/3.0/">
                                     <img alt="Creative Commons licence" data-src="./img/cc-by-sa-3.0-88x31.png">
                                 </a>
                             </div>
 
-                            <div style="display: inline-block; margin: 0 .5em 0 .5em">
+                            <div class="display-inline halfEmMargin">
```

Comment:
Classes shouldn't be made for super specific situations like `. halfEmMargin`, prefer `id`s in this case.

Reply:
Decent rule of thumb: if you're not gonna use it again, probably not a good candidate for a class.